fdbclient/BlobCipher.cpp

```diff
@@ -54,6 +56,42 @@ bool isEncryptHeaderAuthTokenModeValid(const EncryptAuthTokenMode mode) {
 }
 } // namespace
 
+// BlobCipherMetrics methods
+
+BlobCipherMetrics::CounterSet::CounterSet(CounterCollection& cc, std::string name)
+  : encryptCPUTimeNS(name + "EncryptCPUTimeNS", cc), decryptCPUTimeNS(name + "DecryptCPUTimeNS", cc),
+    getCipherKeysLatency(name + "GetCipherKeysLatency",
+                         UID(),
+                         FLOW_KNOBS->ENCRYPT_KEY_CACHE_LOGGING_INTERVAL,
+                         FLOW_KNOBS->ENCRYPT_KEY_CACHE_LOGGING_SAMPLE_SIZE),
+    getLatestCipherKeysLatency(name + "GetLatestCipherKeysLatency",
+                               UID(),
+                               FLOW_KNOBS->ENCRYPT_KEY_CACHE_LOGGING_INTERVAL,
+                               FLOW_KNOBS->ENCRYPT_KEY_CACHE_LOGGING_SAMPLE_SIZE) {}
+
+BlobCipherMetrics::BlobCipherMetrics()
+  : cc("BlobCipher"), cipherKeyCacheHit("CipherKeyCacheHit", cc), cipherKeyCacheMiss("CipherKeyCacheMiss", cc),
+    cipherKeyCacheExpired("CipherKeyCacheExpired", cc), latestCipherKeyCacheHit("LatestCipherKeyCacheHit", cc),
+    latestCipherKeyCacheMiss("LatestCipherKeyCacheMiss", cc),
+    latestCipherKeyCacheNeedsRefresh("LatestCipherKeyCacheNeedsRefresh", cc),
+    getCipherKeysLatency("GetCipherKeysLatency",
+                         UID(),
```

Comment:
may be create a `UID` per BlobCipherMetrics instance and use it rather than creating new per `Latency` counter construction?

Reply:
`UID()` is just the `UID(0,0)`. Since BlobCipherMetrics is mean to be a singleton, I didn't assign id to the latency samples.